src/proximity-manager/index.ts

```diff
@@ -58,13 +56,18 @@ const ProximityManager = () => {
   // a temporary buffer is used to store the chunks of the message
   let tempBuffer: number[] = [];
 
+  // a random UUID is used to identify the mDL-reader
+  // it should be generated at the first start of the manager
+  // and updated on every restart
+  let randomVerifierUUID: string;
+
   const eventManager = createEventManager();
 
   const start = () => {
     // TODO: generate keypair
-    return new Promise<void>((resolve, reject) => {
-      // TODO: this shouldn't be called multiple times
-      // add a method to check if it is already started
```

Comment:
If possibile it should be better to not remove this comment. What do you think about leaving the comment and add this jira task https://pagopa.atlassian.net/browse/SIW-708?atlOrigin=eyJpIjoiYjY2MzAxOWIxZWYzNDk0ODgzN2VhOGM3YjgyNzBhZjgiLCJwIjoiaiJ9

Reply:
I don't know if the session alone would be sufficient to track the status of BLEManager given the independent nature of the module. However we have a task in Jira to remind to add this check.